views/index.hbs

```diff
@@ -1,2 +1,6 @@
+
```

Comment:
You could also use an anchor element `<a href="/drones">Check for drones available</a>`, since you're not passing any data with your request I feel using a form element isn't necessary.